graphql/query_resolver.go

```diff
@@ -552,14 +552,14 @@ func (r *queryResolver) MyHosts(ctx context.Context) ([]*restModel.APIHost, erro
 	hosts, err := host.Find(ctx, host.ByUserWithRunningStatus(usr.Username()))
 	if err != nil {
 		return nil, InternalServerError.Send(ctx,
-			fmt.Sprintf("Error finding running hosts for user %s : %s", usr.Username(), err))
+			fmt.Sprintf("finding running hosts for user '%s' : %s", usr.Username(), err))
```

Comment:
this seems like maybe it was a typo in the original implementation
```suggestion
			fmt.Sprintf("finding running hosts for user '%s' : %s", usr.Username(), err.Error()))
```